ingredient/ingredient-functions/src/arm.json

```diff
@@ -0,0 +1,129 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "funcAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique name of the function app to create or update."
+            }
+        },
+        "funcAppResourceGroup": {
+            "type": "string",
+            "metadata": {
+                "description": "App Service resource group. If not set, it defaults to the resource group of the Bake deployment context."
+            }
+        },
+        "storageAccountConnectionString": {
+            "type": "string",
+            "metadata": {
+                "description": "Connection string of the storage account."
+            }
+        },
+        "container_registry_url": {
+            "type": "string",
+            "metadata": {
+                "description": "Docker container registry address."
+            }
+        },
+        "container_registry_user": {
+            "type": "string",
+            "metadata": {
+                "description": "Docker container registry user name."
+            }
+        },
+        "container_registry_password": {
+            "type": "string",
+            "metadata": {
+                "description": "Docker container registry user password."
+            }
+        },
+        "container_image_name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Docker Linux-based image to deploy. This image should contain the function app artifacts."
+            }
+        },
+        "hostingPlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the App Service hosting plan."
+            }
+        },
+        "location": {
```

Comment:
for location, set a default value here that gets the location of the resource group of the deployment (look at some of our other ARM templates that you can copy/paste from.